meta-dts-distro/recipes-kernel/kernel/linux-yocto/intel-mei.cfg

```diff
@@ -0,0 +1,5 @@
+CONFIG_INTEL_MEI=y
+CONFIG_INTEL_MEI_ME=y
+CONFIG_INTEL_MEI_TXE=y
+CONFIG_INTEL_MEI_HDCP=m
```

Comment:
@kotylamichal is this module loaded after boot?

Reply:
@TomaszAIR no, it is not used. I will remove that